app/src/main/res/layout-sw600dp-port/profile_chooser_fragment.xml

```diff
@@ -43,8 +43,7 @@
           app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
-          style="@style/Subtitle1"
-          android:gravity="center"
+          style="@style/Subtitle1ViewCenterVertical"
```

Comment:
Given that android:gravity="center" is both horizontal & vertical, do we need a style that sets both for these cases?

Reply:
Yes added `Subtitle1Center` style.